src/CssVariables/index.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
```

Comment:
We have moved from the directory structure of `src/{Component}/index.js` to `src/{Component}.js` in `next`. So either we merge this to `master` as is, and then refactor it immediately for `next`, or we use `src/CssVariables.js` in this PR. The discrepancy internally doesn't matter (in this case) since it is exported from `src/index.js` anyway.

Reply:
I'd prefer `src/CssVariables.js`, it will cause less work (no need for another PR).
Also `next` is soon to be merged anyways, so it doesn't matter if there's a bit on inconsistency internally for a short amount of time. The users of this lib won't notice it

Reply:
All right, that's fine for me as well. @varl?

Reply:
Sure.

Reply:
Done, should be good to go now 👍